modules/local/sra_fastq_ftp.nf

```diff
@@ -29,19 +30,22 @@ process SRA_FASTQ_FTP {
     tuple val(meta), path("*md5")     , emit: md5
 
     script:    
+    def args  = options.args  ? options.args  : '-C - --max-time 600'
+    def args2 = options.args2 ? options.args2 : '12h'

```

Comment:
Why do we need these lines to reset `options.args` and `options.args2`? Can just provide one or the other in `modules.config` by default?

Reply:
I just like the module being self contained with defaults in case it ends up somewhere else but happy to delete if its not the standard.

Reply:
Yes please :) We should only be defining mandatory arguments in modules to keep them as flexible as possible. All non-mandatory options should be coming from `options`